build.gradle

```diff
@@ -927,69 +425,35 @@ publishing {
 			}
 			artifact sourcesJar
 			artifact javadocJar
-		}
 
+			pom.withXml {
+				def hashedDep = it.asNode().appendNode('dependencies').appendNode('dependency')
+				hashedDep.appendNode('groupId', 'org.quiltmc')
+				hashedDep.appendNode('artifactId', 'hashed')
+				hashedDep.appendNode('version', configurations.hashed.getDependencies().iterator().next().getVersion())
+			}
+		}
 	}
 	repositories {
-		if (ENV.MAVEN_URL) {
+		if (ENV.MAVEN_URL && !USE_SNAPSHOT_HASHES) {
```

Comment:
This means QM will be published to the snapshot maven while we use snapshot hashed? I'd keep it in the release maven